src/core/net/nat64_translator.hpp

```diff
@@ -328,35 +339,63 @@ class Translator : public InstanceLocator, private NonCopyable
 
         typedef String<Ip6::Address::kInfoStringSize + Ip4::Address::kAddressStringSize + 4> InfoString;
 
-        void       Touch(TimeMilli aNow) { mExpiry = aNow + kAddressMappingIdleTimeoutMsec; }
+        struct ip6MapSearch
+        {
+            Ip6::Address ip6Addr;
+            uint16_t     srcPortOrId;
```

Comment:
Use `m` prefix for the member variables.
```suggestion
            Ip6::Address mIp6Addr;
            uint16_t     mSrcPortOrId;
```